src/test/java/guitests/StartupBoardLauncherTest.java

```diff
@@ -0,0 +1,47 @@
+package guitests;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.loadui.testfx.utils.FXTestUtils;
+import prefs.Preferences;
+import ui.BoardAutoCreator;
+import ui.TestController;
+import ui.UI;
+import ui.issuepanel.PanelControl;
+import static guitests.BoardAutoCreatorTest.verifyBoard;
+
+import static org.junit.Assert.assertEquals;
+import static ui.BoardAutoCreator.SAMPLE_BOARD;
+
+public class StartupBoardLauncherTest extends UITest{
+
+    private PanelControl panelControl;
+    private Preferences testPref;
```

Comment:
By the same logic, the previous variable should be called `testPanelControl`? Is it a preference object specifically created for testing? If that is the case, it can be called `testPref`. If it is a regular preference object that happens to be used during testing, then I don't see why it is prefixed with `test` when other variables are not.